docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -47,34 +95,61 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -Dextensions="resteasy,spring-di"
 cd spring-di-quickstart
 ----
+////
 
-This command generates a Maven project with a REST endpoint and imports the `spring-di` extension.
+//Alternative 2: you have a project and need to add the required extension.
+[id="setting-up-your-spring-di-maven-project_{context}"]
+== Setting up your Spring DI example Maven project
 
-If you already have your Quarkus project configured, you can add the `spring-di` extension
-to your project by running the following command in your project base directory:
+[role="_abstract"]
+Add the required {ProductName} extension as a dependency of your Maven project.
 
+.Procedure
+
+. Navigate to the the top-level directory of your {ProductName} Maven project:
++
+[source,bash,options="nowrap",subs="+quotes,attributes+"]
+----
+cd <name-of-your-project-directory>
```

Comment:
maybe consider `$ cd` to make it obvs it's run a terminal without sudo permissions. if sudo, then `#`

Reply:
@oraNod Good point! Currently, neither the upstream nor the downstream docs seems to use the PS shell symbol consistently in command examples across guides. I can introduce the change at the risk of making the changes slightly more invasive.